app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx

```diff
@@ -310,6 +311,11 @@ export class Execution extends React.PureComponent<IExecutionProps, IExecutionSt
         key={stage.refId}
         application={application}
         execution={execution}
+        manualJudgment={
+          this.props.manualJudgment !== undefined && this.props.manualJudgment[this.props.execution.id]
+            ? this.props.manualJudgment
+            : []
+        }
```

Comment:
I would pull out manual judgement on line 290-299. Execution is already defined so you don't have to repeat `this.props`.